### DISCUSSION TITLE
High idle CPU usage

### DISCUSSION THREAD

**Discussion by aneeshjoy** [itzg/docker-minecraft-bedrock-server#312] [Q&A]
I setup this awesome docker image for my kids on my home lab server. Noticed consistently high cpu usage which I know  will eventually end up in my electricity bill over a period of time if i run 24x7.

Is there any way to tune this down to near zero?
Thanks in advance for any help or pointer's.

Attaching a few screenshots from portainer  ui.

![Screenshot_20230409-091143~2](https://user-images.githubusercontent.com/5285961/230762979-dacf8db9-af40-46b8-80e0-53aa19603670.jpg)
![Screenshot_20230409-090823](https://user-images.githubusercontent.com/5285961/230762980-300ce544-6327-4f9c-bde5-ff383d8e5eb2.jpg)

**Comment by itzg** [comment]
Maybe I'm reading this wrong, but I only see 2% cpu utilization. Where is the high cpu usage you speak of?

Be sure to measure your electricity usage with an outlet device to ensure you're worrying about something worth worrying about.

**Reply by aneeshjoy** [reply]
You are right its near 2%. It stays at this level all the time even when no one is playing. 

If I can find out what its doing when no one is playing on the server, I could just turn it off and save some cpu cycles.  logging, health check etc.

I could shut it down after use. But always on is cool if its not using much cpu.

**Comment by aneeshjoy** [comment]
It seems to be the healthcheck. How to turn it off or reduce frequency?

![Screenshot_20230409-144425](https://user-images.githubusercontent.com/5285961/230776387-51f28f89-f47a-464b-a4af-5a65177ef494.jpg)

**Reply by itzg** [reply]
I'm not familiar with Portainer (I dislike UI management of Docker/K8S) so you'll need to look for instructions on configuring the health check. 

Again, is this really something to worry about? Is it 2% CPU? What is your overall computer CPU usage? Containers are just Linux processes like anything else.

**Reply by itzg** [reply]
(BTW, I hope I don't sound dismissive. I'm a big believer in **mental** energy conservation -- don't worry about things that are not worth worrying about)

**Comment by aneeshjoy** [comment]
Absolutely not something to worry about. Thanks for responding.
